templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/terraform/linuxvm.tf

```diff
@@ -131,6 +131,7 @@ data "template_file" "apt_sources_config" {
   template = file("${path.module}/apt_sources_config.yml")
   vars = {
     nexus_proxy_url = local.nexus_proxy_url
+    sku             = local.apt_sku
```

Comment:
I am concerned that the inconsistency between using `apt`, `apt-sku` and `apt_sku` is going to make maintenance harder.

Reply:
agreed, I'll change it to something more consistent